src/DerConverter/DerConverter.csproj

```diff
@@ -13,7 +13,7 @@
     <PackageLicenseUrl>https://raw.githubusercontent.com/huysentruitw/pem-utils/master/LICENSE</PackageLicenseUrl>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/huysentruitw/pem-utils</RepositoryUrl>
-    <TargetFrameworks>net471;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net451;netstandard2.0</TargetFrameworks>
```

Comment:
Add net451 as an additional target. Dropping explicit support for net471 can cause weird side effects to projects using the NuGet package. See https://stackoverflow.com/questions/48060230/system-reference-troubles-when-introducing-netstandard-library-dependency